rules/jsonschema-draft7-to-2019-09.json

```diff
@@ -62,7 +73,9 @@
         "$schema": "https://json-schema.org/draft/2020-12/schema",
         "$id": "https://github.com/sourcemeta/alterschema/rules/jsonschema-draft7-to-2019-09/definitions",
         "type": "object",
-        "required": [ "definitions" ],
+        "required": [
```

Comment:
All of these look like irrelevant formatting changes. Can we omit them from this PR? If we want to reformat, let's do it on a separate PR purely dedicated to that